workflows/diagnostics/tests/prognostic/emulation/test_single_run.py

```diff
@@ -265,9 +265,16 @@ def test_skill_table(regtest):
         print(name, ":", signature, file=regtest)
 
 
-def test_compute_summaries(regtest):
+@pytest.mark.parametrize(
+    "func",
+    [
+        pytest.param(func, id=func.__name__)
+        for func in single_run.get_summary_functions()
+    ],
+)
+def test_compute_summaries(func, regtest):
```

Comment:
Rename this to `test_get_summary_functions`?

Reply:
renamed to `test_summary_function` since that is what `func` is.